<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slicePainter</class>
 <widget class="QWidget" name="slicePainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>340</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>256</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="slice">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sliderLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QSlider" name="sliceSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdSlice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="numDigits">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Initialization Method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="drawVertebraOutlineRadioButton">
        <property name="text">
         <string>Draw Vertebra Outline</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="placeSinglePointRadioButton">
        <property name="text">
         <string>Place Single Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="placeCenterAndTwoBPRadioButton">
        <property name="text">
         <string>Place Center + Two Boundary Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pickVertebraCentersRadioButton">
        <property name="text">
         <string>Pick Vertebra Centers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pickThreePointsRadioButton">
        <property name="text">
         <string>Pick Vertebra, Disk and Spinal Cord Points</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="startGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Bottom-most vertebra:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bottomLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>S5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdSlice</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
